### PR TITLE
Add Czech translations for Cura 4.8

### PR DESCRIPTION
**Pull request by sustmi**
Add missing Czech translations and re-enable the option to switch to Czech language.
Fixes https://github.com/Ultimaker/Cura/issues/8821 .

### CODE REVIEW

File: resources/i18n/cs_CZ/cura.po [hunk 1]
```diff
@@ -1440,15 +1440,23 @@ msgid_plural ""
 "You are about to remove {0} printers from Cura. This action cannot be undone.\n"
 "Are you sure you want to continue?"
 msgstr[0] ""
+"Chystáte se odebrat {0} tiskárnu from Cury. Tuto akci nelze vrátit zpět.\n"
```
Comment (konvoj) [inline resources/i18n/cs_CZ/cura.po:1443]: "Chystáte se odebrat {0} tiskárnu z Cury. Tuto akci nelze vrátit zpět.\n"

File: resources/i18n/cs_CZ/cura.po [hunk 2]
```diff
@@ -1440,15 +1440,23 @@ msgid_plural ""
...
+"Doopravdy chcete pokračovat?"
 msgstr[1] ""
+"Chystáte se odebrat {0} tiskárny from Cury. Tuto akci nelze vrátit zpět.\n"
```
Comment (konvoj) [inline resources/i18n/cs_CZ/cura.po:1446]: "Chystáte se odebrat {0} tiskárnu from Cury. Tuto akci nelze vrátit zpět.\n"

File: resources/i18n/cs_CZ/cura.po [hunk 3]
```diff
@@ -1440,15 +1440,23 @@ msgid_plural ""
...
+"Doopravdy chcete pokračovat?"
 msgstr[2] ""
+"Chystáte se odebrat {0} tiskáren from Cury. Tuto akci nelze vrátit zpět.\n"
```
Comment (konvoj) [inline resources/i18n/cs_CZ/cura.po:1449]: "Chystáte se odebrat {0} tiskárnu from Cury. Tuto akci nelze vrátit zpět.\n"

File: resources/i18n/cs_CZ/cura.po (hunk 1 above)
Comment (sustmi) [reply resources/i18n/cs_CZ/cura.po]: Oops. Fixed. Thank you for noticing. :+1:

**Comment by Ghostkeeper** [comment]
This was laid to rest for a while. Sorry about that.
The translation was submitted after the release of 4.8. It would update the translations for the next release, and that is fine. However for the next release we'll again have new strings to translate.

Would it be okay if I e-mail you with an updated translation template when we have a string freeze for the next release? It'll be a template with ~50 or so new texts that need to be translated for a new release. That's the only way we can ensure a complete translation for the next release.
I don't have a date for the next release yet, but it'll be at least a month yet before we release the beta I think.

**Comment by sustmi** [comment]
Yeah, that would be OK.

**Comment by sustmi** [comment]
I think you already have my e-mail address, don't you?
You can even just let me know when the translations are freezed and tell me in which branch/commit I can find the strings to translate.

**Comment by Ghostkeeper** [comment]
Yeah, I grabbed your e-mail address from the commit that you made here. The e-mail address I found ends with `@****rum.cz` (it's public knowledge since your commit is public, but I censored it to not make it any easier on any bots that might look around here).

I expect that we'll freeze the strings for 4.9 in a month or so. You'll get complete instructions then as to which branch to look at and which files to fill in!

**Pull request merged**